ui/apps/platform/cypress/integration/vulnerabilities/nodeCves/nodeCvesOverviewPage.test.ts

```diff
@@ -68,25 +84,135 @@ describe('Node CVEs - Overview Page', () => {
+                aggregateBy: { aggregateFunc: 'max', distinct: false },
+            })
+        );
+        interactAndInspectGraphQLVariables(() => sortByTableHeader('Top CVSS'), 'getNodeCVEs').then(
+            expectRequestedSort({
+                field: 'CVSS',
+                reversed: false,
+                aggregateBy: { aggregateFunc: 'max', distinct: false },
+            })
+        );
+
         // check sorting of Affected Nodes column
-        // check sorting of First discovered column
+        interactAndInspectGraphQLVariables(
+            () => sortByTableHeader('Affected Nodes'),
+            'getNodeCVEs'
+        ).then(
+            expectRequestedSort({
+                field: 'Node ID',
+                reversed: true,
+                aggregateBy: { aggregateFunc: 'count', distinct: true },
+            })
+        );
+        interactAndInspectGraphQLVariables(
+            () => sortByTableHeader('Affected Nodes'),
+            'getNodeCVEs'
+        ).then(
+            expectRequestedSort({
+                field: 'Node ID',
+                reversed: false,
+                aggregateBy: { aggregateFunc: 'count', distinct: true },
+            })
+        );
+
+        // check that the First Discovered column is not sortable
+        queryTableHeader('First Discovered');
+        queryTableSortHeader('First Discovered').should('not.exist');
     });
 
     it('should sort Node table columns', () => {
+        visitNodeCveOverviewPage();
+
+        // Visit Node tab and wait for initial load - sorting will be pre-applied to the Node column
+        interactAndWaitForResponses(
+            () => cy.get(vulnSelectors.entityTypeToggleItem('Node')).click(),
+            getRouteMatcherMapForGraphQL(['getNodes'])
+        );
+
         // check sorting of Node column
-        // check sorting of CVEs by Severity column
+        interactAndInspectGraphQLVariables(() => sortByTableHeader('Node'), 'getNodes').then(
+            expectRequestedSort({ field: 'Node', reversed: true })
+        );
+        interactAndInspectGraphQLVariables(() => sortByTableHeader('Node'), 'getNodes').then(
+            expectRequestedSort({ field: 'Node', reversed: false })
+        );
+
+        // check that CVEs by Severity is not sortable
+        queryTableHeader('CVEs by severity');
+        queryTableSortHeader('CVEs by severity').should('not.exist');
+
         // check sorting of Cluster column
+        interactAndInspectGraphQLVariables(() => sortByTableHeader('Cluster'), 'getNodes').then(
+            expectRequestedSort({ field: 'Cluster', reversed: true })
+        );
+        interactAndInspectGraphQLVariables(() => sortByTableHeader('Cluster'), 'getNodes').then(
+            expectRequestedSort({ field: 'Cluster', reversed: false })
+        );
+
         // check sorting of Operating System column
+        interactAndInspectGraphQLVariables(
+            () => sortByTableHeader('Operating System'),
```

Comment:
As commented above, what are pro and con:
* **Operating system** sentence case above according to UI
* **Operating System** title case below according to API

A test scenario above it a counter-example to these similar strings where the strings differ:
* **Affected nodes** sentence case according to UI
* **Node ID** title case according to API